src/Components/Endpoints/test/Microsoft.AspNetCore.Components.Endpoints.Tests.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I'm sure this project will be successful across the board 😁:

Reply:
I think he's just remarking that it's a completely empty test project so it's always going to pass :)

Reply:
> I think he's just remarking that it's a completely empty test project so it's always going to pass :)

exactly

Reply:
It has a couple of tests now 😄.

This PR is only adding all the scaffold for the things we need.